core/src/main/java/org/apache/calcite/sql/dialect/SnowflakeSqlDialect.java

```diff
@@ -109,6 +109,14 @@ private SqlOperator getTargetFunctionForDateOperations(RexCall call) {
     case TRIM:
       unparseTrim(writer, call, leftPrec, rightPrec);
       break;
+    case IF:
+      final SqlWriter.Frame iffFrame = writer.startFunCall("IFF");
+      for (SqlNode operand : call.getOperandList()) {
+        writer.sep(",");
+        operand.unparse(writer, leftPrec, rightPrec);
+      }
+      writer.endFunCall(iffFrame);
+      break;
```

Comment:
IFF support was already added, is there any new change?

Reply:
IFF function already defined but SnowFlake couldn't find IFF support. Now IF is replaced by IFF.